setup.py

```diff
@@ -1007,7 +1007,7 @@ def _property_entry_points(self):
                             'salt-key = salt.scripts:salt_key',
                             'salt-master = salt.scripts:salt_master',
                             'salt-minion = salt.scripts:salt_minion',
-                            'salt-unity = salt.scripts:salt_unity',
+                            'salt-unity = salt.unity:salt_unity',
```

Comment:
I do not like making this change.  Because now they are not consistent across all of the different scripts.

Reply:
I don't either, but my python-fu is not good enough to port the `get_avail` function ( https://github.com/saltstack/salt/blob/develop/scripts/salt-unity#L11 ) to instrospect from within `scripts.py`

Is it possible for a function inside a module to list all functions defined in that module?
I don't think so which I think is why those functions live inside the script file instead of in `scripts.py`.

The current `setup.py` code, while pretty, is broken. My code, while less pretty, works.

If there's a way to write pretty code that works, I'm all for it, I just don't know the way forward to doing that.

Reply:
Actually, I may be able to use `inspect` to do this.

Reply:
Ok, I am not going to approve this with it being moved.

What is the actual problem you are trying to solve? What isn't working?